CircuitVerse/CircuitVerse
# Integration testing for profile management

#### Describe the changes you have made in this PR -
Test cases added for 
  - edit name
  - edit country
  - edit educational institute
  - edit locale
  
Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.
